source/ptreferential/ptreferential.cpp

```diff
@@ -275,9 +280,83 @@ std::vector<idx_t> manage_odt_level(const std::vector<type::idx_t>& final_indexe
     return final_indexes;
 }
 
+bool keep_vj(const nt::VehicleJourney* vj,
```

Comment:
a very general comment: functions like these, which are details of implementation, not exposed in header and only used in this .cpp should be put in an unnamed namespace. :) it's ok for me that nothing to do in this PR